tests/unit/utils/test_rsax931.py

```diff
@@ -218,6 +214,28 @@ def test_find_libcrypto_darwin_catalina(self):
                 break
         self.assertTrue(passed)
 
+    @skipIf(not salt.utils.platform.is_darwin(), "Host OS is not Darwin-like or macOS.")
+    @patch.object(platform, "mac_ver", lambda: ("11.0", (), ""))
+    def test_find_libcrypto_darwin_bigsur(self):
+        """
+        Test _find_libcrypto on a Darwin-like  macOS host where there isn't a
+        lacation returned by ctypes.util.find_library()
+        """
+        lib_path = _find_libcrypto()
+        brew_prefix = os.getenv("HOMEBREW_PREFIX", "/usr/local")
+        passed = False
+        for i in (
+            "/opt/salt/lib/libcrypto.dylib",
+            os.path.join(brew_prefix, "opt/openssl/lib/libcrypto.dylib"),
+            os.path.join(brew_prefix, "opt/openssl@*/lib/libcrypto.dylib"),
+            "/opt/local/lib/libcrypto.dylib",
+            "/usr/lib/libcrypto.*.dylib",
+        ):
+            if fnmatch.fnmatch(lib_path, i):
+                passed = True
+                break
+        self.assertTrue(passed)
```

Comment:
**suggestion**: this test looks like it's simply re-implementing the code it's supposed to be testing. That's not very meaningful.

It would be more meaningful to do something like:

```
def test_when_homebrew_prefix_is_set_on_mac_lib_crypto_should_use_that_path():
    expected_lib_path = "/some/path/to/opt/openssl/lib/libcrypto.dylib"
    # patch getenv to return '/some/path/to'. Also probably need to patch glob
    lib_path = _find_libcrypto()
    assert lib_path == expected_lib_path
```

I would write a couple of tests - at *minimum* to cover the two scenarios that add `BREW_PREFIX`.

It's also preferred to add tests to `tests/pytests/`. If you need help, you can attend one of the [Test Clinics on Twitch](https://www.twitch.tv/saltprojectoss/schedule) and we'll help you get pointed in the right direction.